athena-elasticsearch/src/main/java/com/amazonaws/connectors/athena/elasticsearch/ElasticsearchMetadataHandler.java

```diff
@@ -83,6 +87,17 @@
     // A Map of the domain-names and their respective endpoints.
     private Map<String, String> domainMap;
 
+    /**
+     * Key used to store shard information in the Split's properties map (later used by the Record Handler).
+     */
+    private static final String SHARD_KEY = "shard";
```

Comment:
probably needs to be protected so that the record handler can reference it?

Reply:
Changed to protected and reused in Record Handler.